fluentd/configs.d/filter/cloud_init.conf

```diff
@@ -0,0 +1,5 @@
+<filter journal>
+  @type grep
+  # drop messages from cloud-init - they are not logged to /var/log/messages by default
+  exclude1 _COMM ^cloud-init$
```

Comment:
So you're saying cloud-init logs are left out of syslog, but go into the journal? I looked at to config to see why that might be true, guessing it's some kind of boot-time weirdness. In any case, it seems odd in my opinion to explicitly exclude something that goes into the journal unless there's a specific problem with it. Cloud-init runs once at boot and I can't see how including its output would cause any problems. Meanwhile, sysadmins might be mystified why it doesn't show up in Kibana and start to distrust the solution.

So it seems to me we should leave cloud-init entries alone.